phpunit/class-wp-theme-json-test.php

```diff
@@ -4466,27 +4466,27 @@ public function data_set_spacing_sizes() {
 				),
 				'expected_output' => array(
 					array(
-						'name' => '1',
+						'name' => 'X-Small',
 						'slug' => '30',
 						'size' => '0.09rem',
 					),
 					array(
-						'name' => '2',
+						'name' => 'Small',
 						'slug' => '40',
 						'size' => '0.38rem',
 					),
 					array(
-						'name' => '3',
+						'name' => 'Medium',
 						'slug' => '50',
 						'size' => '1.5rem',
 					),
 					array(
-						'name' => '4',
+						'name' => 'Large',
 						'slug' => '60',
 						'size' => '6rem',
 					),
 					array(
-						'name' => '5',
+						'name' => 'X-Large',
```

Comment:
These renames are all done in the UI instead in case the UI wants to show the numbers in the tooltips in the slider, but an actually useful name elsewhere. See https://github.com/WordPress/gutenberg/pull/44247.